src/components/common/ModalTutorial.vue

```diff
@@ -40,6 +40,16 @@
               >
                 {{ item }}
               </span>
+              <span
```

Comment:
you added this for all tutorials. shouldn't it only appear in the "get tokens" tutorial?

Reply:
that's why I have the `v-if="index === 0"` in place

Reply:
noooooo
ModalTutorial is used for all tutorials like governance. The link should only appear in the "get tokens" tutorial....

Reply:
Oh no!

Reply:
Fixed ✌️

Reply:
Can you explain what you are trying to do here?

Reply:
Salvation! Implements logic for further affiliate links within the `affiliate` property 💃 

Cannot ask for more in one single PR (that originally just aimed to include a simple link) 😉